WordPress/src/main/res/values/strings.xml

```diff
@@ -1689,6 +1689,7 @@
     <string name="reader_empty_posts_liked_description">Posts that you like will appear here</string>
     <string name="reader_empty_saved_posts_title">No posts saved — yet!</string>
     <string name="reader_empty_saved_posts_description">Tap %s to save a post to your list.</string>
+    <string name="reader_empty_saved_posts_content_description">Tap the bookmark icon to save a post to your list.</string>
```

Comment:
I was thinking that “bookmark button” would be more appropriate. But when I tried that button with TalkBack, it says “Add to Save Posts button”. Perhaps that's what we should use?

Reply:
Good point. Yes, I was thinking to make the other bookmark button but since it's already "Add To Save Posts" it's better to leave it at that & it's very descriptive as well. Thanks for catching this 👍

Reply:
Resolved here https://github.com/wordpress-mobile/WordPress-Android/pull/10741/commits/66ccd1b767ccbe8e0abbb7e3aba8c0ded293c16e